src/MICore/Debugger.cs

```diff
@@ -49,6 +49,8 @@ public class Debugger : ITransportCallback
 
         public bool IsCygwin { get; protected set; }
 
+        public bool IsLocalGdbOnWindows { get; protected set; }
```

Comment:
@gregg-miskelly What cases would that be? I'd be ok with renaming it. Bash on Windows uses pipe so it isn't going through this path and I can't think of another "local" launch that uses gdb that wouldn't want this.